packages/reports/addon/templates/components/common-actions/schedule.hbs

```diff
@@ -9,105 +9,73 @@
-              @onToggle={{toggle "mustHaveData" this.localDeliveryRule.schedulingRules}}
-            />
-            <div class="schedule-modal__label">Only send if data is present</div>
-          </div>
-        {{/if}}
-        <div class="schedule-modal__btn-container btn-container">
-          <div class="button-group">
+          <DenaliInputGroup class="m-b-30" @label="Frequency" as |Group|>
+              <Group.Select
+                class="schedule-modal__dropdown schedule-modal__dropdown--frequency"
+                @options={{this.frequencies}}
+                @selectedOption={{this.localDeliveryRule.frequency}}
+                @onChange={{fn (mut this.localDeliveryRule.frequency)}}
+                disabled={{is-rejected this.deliveryRule}}
+                as | frequency |
+              >
+                {{capitalize frequency}}
+              </Group.Select>
+          </DenaliInputGroup>
+          <DenaliInputGroup class="m-b-30" @label="Format" as |Group|>
+              <Group.Select
+                class="schedule-modal__dropdown schedule-modal__dropdown--format"
+                disabled={{lt this.formats.length 2}}
+                @options={{this.formats}}
+                @selectedOption={{this.localDeliveryRule.format.type}}
+                @onChange={{this.updateFormat}}
+                as | format |
+              >
+                {{format}}
+              </Group.Select>
+          </DenaliInputGroup>
+          {{#if (feature-flag "enabledNotifyIfData")}}
+            <DenaliInputGroup class="m-b-30" @label="Only send if data is present" as |Group|>
+              <Group.Switch
+                class="schedule-modal__must-have-data-toggle"
+                @isEnabled={{this.localDeliveryRule.schedulingRules.mustHaveData}}
+                @onChange={{toggle "mustHaveData" this.localDeliveryRule.schedulingRules}}
+              />
+            </DenaliInputGroup>
+          {{/if}}
+          <div class="schedule-modal__btn-container btn-container">
             {{#if (is-rejected this.deliveryRule)}}
-              <button
-                class="schedule-modal__cancel-btn btn btn-primary"
+              <DenaliButton
+                class="schedule-modal__cancel-btn button is-solid btn-primary m-5"
                 disabled={{this.isSaving}}
                 type="button"
                 {{on "click" (queue this.closeModal (fn this.onRevert this.localDeliveryRule))}}
               > Cancel
-              </button>
+              </DenaliButton>
             {{else}}
-              <button
-                class="schedule-modal__save-btn btn btn-primary"
+              <DenaliButton
+                class="schedule-modal__save-btn m-5"
                 disabled={{this.disableSave}}
-                type="button"
                 {{on "click" (queue (toggle "isSaving" this) this.doSave)}}
               >
                 {{#if this.isSaving}}
```

Comment:
This portion does not show up nicely inside the button

Reply:
Think this is due to it using a NaviLoader, should I try and fix that in this pr, or just wait until my navi loader pr which should be later this week?